client/src/containers/CampaignAdmin/CampaignAdmin.js

```diff
@@ -79,12 +80,12 @@ class CampaignAdmin extends Component {
   }
 
   setBreadcrumbs(view, id, title) {
-    const { sectionConfig: { url } } = this.props;
+    const { sectionConfig: { reactRoutePath } } = this.props;
 
     // Set root breadcrumb
     const breadcrumbs = [{
       text: i18n._t('CampaignAdmin.CAMPAIGN', 'Campaigns'),
-      href: url,
+      href: `/${reactRoutePath}`,
```

Comment:
Can we always guarantee that reactRoutePath won't have a leading slash?

Reply:
Theoretically yes but realistically there's the remote chance someone will do something really weird lol. I'll apply `joinUrlPaths` to it as you recommended in another comment.

Reply:
Done